far/dirinfo.cpp

```diff
@@ -111,40 +111,44 @@ int GetDirInfo(string_view const DirName, DirInfoData& Data, multifilter* Filter
 	// Временные хранилища имён каталогов
 	string strLastDirName;
 	os::fs::find_data FindData;
+
+	// Mantis#0002692
+	struct SuspendMacros
+	{
+		SuspendMacros() { Global->CtrlObject->Macro.SuspendMacros(true); }
+		~SuspendMacros() { Global->CtrlObject->Macro.SuspendMacros(false); }
+	} Suspend;
+
 	while (ScTree.GetNextName(FindData,strFullName))
 	{
-		// Mantis#0002692
-		if (!Global->CtrlObject->Macro.IsExecuting())
-		{
-			INPUT_RECORD rec;
+		INPUT_RECORD rec;
```

Comment:
Can we keep the extra indentation of this block just to reduce the diff size and make it more obvious what exactly has been changed?

We can fix source formatting in one go later (we need to do it anyway).

Reply:
I mean we can fix it for all the repo later in one commit.
It's ok to leave it as is now, i.e. without 0c189c91a58389591d62787353fba2927d28cab1.

Reply:
@alabuzhev 
Done